tracer/test/snapshots/Iast.Tbv.AspNetMvc5.IastDisabled.verified.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
An empty file seems wrong, shouldn't there be spans but without the IAST info?

Reply:
I changed the name of the test. I'm removing this